controllers/TasksController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace app\controllers;
+
+use app\models\Task;
+
+class TasksController extends \yii\web\Controller
+{
+    public function actionIndex()
+    {
+        $tasks = Task::find()
+            ->where(['task_status' => '0'])
```

Comment:
В коде никогда не должно быть непонятных чисел, строк и других подобных переменных.
Во-первых, мне, как новому программисту, который пришёл в проект сложно будет понять такую запись. Выбираются задания со статусом 0, но что этот статус значит - не ясно. Как пример, в котором ничего не понятно:
```php
$data = Model::find()
    ->where(['IN', 'status', [1, 5, 27, 28]])
    ->andWhere(['group' => 54])
    ->andWhere(['vendor_type' => 'C'])
```
Во-вторых, выборки по статусу будут не только в контроллере, но и в моделях и других классах. В учебном проекте такого нет, но в живых коммерческих проектах всегда будут вноситься изменения. И если, к примеру, числовое значение статуса решат изменить, то при таком подходе придётся искать по всему проекту, где написано `0`. При использовании констант, достаточно будет изменить только её значение в одном месте

```suggestion
            ->where(['task_status' => Task::STATUS_NEW])
```

В класс `Task` нужно будет добавить соответствующую константу